components/site-header.tsx

```diff
@@ -47,15 +56,17 @@ export async function SiteHeader({ user }: SiteHeaderProps) {
             className={cn(
               buttonVariants(),
               "hidden max-w-52 gap-2 overflow-hidden whitespace-pre md:flex",
-              "group relative w-full justify-center gap-2 rounded-md transition-all duration-300 ease-out hover:ring-2 hover:ring-primary hover:ring-offset-2",
+              "group relative w-full justify-center gap-2 rounded-md transition-all duration-300 ease-out hover:ring-2 hover:ring-primary hover:ring-offset-2"
             )}
             target="_blank"
             href={siteConfig.links.github}
           >
             <span className="absolute right-0 -mt-12 h-32 w-8 translate-x-12 rotate-12 transform bg-white opacity-10 transition-all duration-1000 ease-out group-hover:-translate-x-40" />
-            <Icons.gitHub className="h-4 w-4" />
-            Star on GitHub
-            <div className="hidden items-center gap-1 text-sm md:flex">
+            <div className="flex items-center"> {/* Added a container div */}
+              <Icons.gitHub className="h-4 w-4" />
+              <span className="ml-1">Star on GitHub</span> {/* Adjusted spacing */}
```

Comment:
awesome! thanks for fixing this.

we could probably remove the comments here since they're not needed within the code base for now :)